### PR TITLE
Update dates helper test

### PR DESCRIPTION
**Pull request by gazayas**
Here are the tests for this pull request
- https://github.com/bullet-train-co/bullet_train-core/pull/182

I deleted some of the assertions because we deleted the following over in there:
```ruby
if local_time(timestamp).to_date == local_time(Time.now).to_date
  "Today at #{display_time(timestamp, custom_time_format)}"
# yesterday?
elsif (local_time(timestamp).to_date) == (local_time(Time.now).to_date - 1.day)
  "Yesterday at #{display_time(timestamp, custom_time_format)}"
else
  # ...
end
```

### CODE REVIEW

**Review by kaspth — APPROVED** [review]

**Comment by jagthedrummer** [comment]
@gazayas If we close https://github.com/bullet-train-co/bullet_train-core/pull/182 in favor of https://github.com/bullet-train-co/bullet_train-core/pull/334 then do we still need this PR?

**Comment by gazayas** [comment]
@jagthedrummer Now that I'm looking over these PRs again, I'm thinking I want to leave https://github.com/bullet-train-co/bullet_train-core/pull/182 and this PR open for the time being. The reason being that, these PRs display time objects as is (i.e. - `"Wed, 02 Aug 2023 10:51:29 +0000"`) whereas https://github.com/bullet-train-co/bullet_train-core/pull/334 changes how the time objects are processed but doesn't change the strings themselves such as "Today at 12:00 P.M." which can be seen in the [dates helper](https://github.com/bullet-train-co/bullet_train-core/blob/main/bullet_train/app/helpers/account/dates_helper.rb), so I want to wait until https://github.com/bullet-train-co/bullet_train-core/pull/334 is merged to see how things play out. Sorry! 🙇‍♂️

**Comment by jagthedrummer** [comment]
No problem @gazayas, that sounds totally reasonable. I'll see what I can do about reviewing and merging https://github.com/bullet-train-co/bullet_train-core/pull/334 and then we can take it from there.

**Comment by jagthedrummer** [comment]
Hey @gazayas I just merged https://github.com/bullet-train-co/bullet_train-core/pull/334. Will be interested to see what you decide about this PR and https://github.com/bullet-train-co/bullet_train-core/pull/182.

File: test/system/dates_helper_test.rb [hunk 1]
```diff
@@ -43,20 +43,12 @@ class DatesHelperTest < ApplicationSystemTestCase
         click_on "Back"
 
         # Assert today's date is displayed correctly.
-        assert page.has_text? "Today at #{time.strftime("%l:%M %p").strip}"
+        assert page.has_text? I18n.l(time, format: :default)
```
Comment (kaspth) [inline test/system/dates_helper_test.rb:46]: It looks like this assertion is failing on CI, but the error message isn't that descriptive, I wonder if we should use Capybara `assert_text` method instead?

```suggestion
        assert_text I18n.l(time, format: :default)
```

**Review by kaspth — APPROVED** [review]
I'm not sure why the test is failing, so I just left some ideas for tuning the assertions.

**Comment by jagthedrummer** [comment]
I suspect that the tests might be failing due to them relying on changes made in https://github.com/bullet-train-co/bullet_train-core/pull/182.

@gazayas, if you can push this branch into the main starter repo (instead of your own fork) then both the starter repo and the `core` repo should be able to find the dependent branches and then hopefully tests will pass on both.

**Comment by gazayas** [comment]
> I suspect that the tests might be failing due to them relying on changes made in [bullet-train-co/bullet_train-core#182](https://github.com/bullet-train-co/bullet_train-core/pull/182).
> 
> @gazayas, if you can push this branch into the main starter repo (instead of your own fork) then both the starter repo and the `core` repo should be able to find the dependent branches and then hopefully tests will pass on both.

@jagthedrummer Sure thing, I'm having some trouble trying to push it directly, so I'll submit a new PR!

**Pull request closed without merging**